src/ReactiveUI/Platforms/netcoreapp1.0/PlatformRegistrations.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MS-PL license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Reactive.Concurrency;
+
+namespace ReactiveUI
+{
+    public class PlatformRegistrations : IWantsToRegisterStuff
+    {
+        public void Register(Action<Func<object>, Type> registerFunction)
+        {
+            RxApp.TaskpoolScheduler = TaskPoolScheduler.Default;
```

Comment:
I'm a little wary of this for a couple of reasons
- I think TaskpoolScheduler should already be configured on non-portable platforms (which covers netcoreapp1.0) e.g. https://github.com/reactiveui/ReactiveUI/blob/develop/src/ReactiveUI/RxApp.cs#L38
- I think the MainThreadScheduler should also get set a default value (I think DefaultScheduler.Instance) if nothing else sets it and I think setting TaskpoolScheduler but not MainThreadScheduler is potentially a bit confusing.

I'd be interested to know if there are problems if this line is omitted, or if it's fine without. I think I'd keep the PlatformRegistrations class however as it adds some value on it's own.

I also think it might be worth wiring up an `IPlatformOperations` even if it just returns null, as I think there are cases it would cause a warning to be logged otherwise (@ghuntley may disagree though...)

Reply:
I think don't worry about removing it everywhere for now, I think separate issue (smaller scope = nicer to review :))

Don't worry about the PlatformOperations then, does look like we shouldn't be hitting it at all :)